Makefile.dune

```diff
@@ -1,22 +1,26 @@
 # -*- mode: makefile -*-
 # Dune Makefile for Coq
 
-.PHONY: help voboot states world apidoc
+.PHONY: help voboot states world watch release apidoc clean bootstrap bootstrap-build
```

Comment:
It seems that adding `watch` `release` and `clean` here is actually a bugfix. Can you push this to another PR?

Reply:
OK, will do.